nautobot/extras/api/serializers.py

```diff
@@ -981,6 +982,44 @@ class JobInputSerializer(serializers.Serializer):
     task_queue = serializers.CharField(required=False, allow_blank=True)
 
 
+class JobMultiPartInputSerializer(serializers.Serializer):
+    """JobMultiPartInputSerializer is a "flattened" version of JobInputSerializer for use with multipart/form-data submissions which only accept key-value pairs"""
+
+    _commit = serializers.BooleanField(required=False, default=None)
+    _schedule_name = serializers.CharField(max_length=255, required=False)
+    _schedule_start_time = serializers.DateTimeField(format=None, required=False)
+    _schedule_interval = ChoiceField(choices=JobExecutionType, required=False)
+    _schedule_crontab = serializers.CharField(required=False, allow_blank=True)
+    _task_queue = serializers.CharField(required=False, allow_blank=True)
```

Comment:
Can you update `nautobot/docs/additional-features/jobs.md#via-the-api` with documentation about how to make an appropriately constructed request in this form?